sdk/compute/Microsoft.Azure.Management.Compute/tests/VMScaleSetTests/VMScaleSetRollingUpgradeTests.cs

```diff
@@ -423,6 +423,78 @@ public void TestVMScaleSetExtensionUpgradeAPIs()
                 }
             }
         }
+
+        /// <summary>
+        /// Covers following Operations:
+        /// Create RG
+        /// Create Storage Account
+        /// Create Network Resources with an SLB probe to use as a health probe
+        /// Create VMScaleSet in rolling upgrade mode
+        /// Verify max surge is set correctly
+        /// </summary>
+        [Fact]
+        [Trait("Name", "TestVMScaleSetRollingUpgradeMaxSurge")]
+        public void TestVMScaleSetRollingUpgradeMaxSurge()
```

Comment:
Note to self, this one still needs a passing test run. Will need to wait until after Thanksgiving break probably.